src/test/java/com/puppycrawl/tools/checkstyle/utils/TokenUtilTest.java

```diff
@@ -277,4 +278,18 @@ public void testForEachChild() {
         assertEquals(secondSibling, firstChild, "Mismatched child node");
     }
 
+    @Test
+    public void testIsOfTypeTrue() {
```

Comment:
it is better to create two separate tests with corresponding names

Reply:
Done!